themes/classic/_dev/js/components/drop-down.js

```diff
@@ -1,8 +1,10 @@
+/* global $ */
```

Comment:
This will work, but only because $ is exposed globally (which it shouldn't, it's just because most jQuery modules suck and haven't heard of the concept of modules yet).

```javascript
import $ from 'jquery';
```